NetCoreRestApi/WebAPI/Startup.cs

```diff
@@ -15,7 +18,9 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
             services.AddTransient<IProductManager, ProductManager>();
-            services.AddTransient<IProductConverter, ProductConverter>();
+            services.AddTransient<IConverter<ProductDTO, ProductModel>, ProductConverter>();
+            services.AddTransient<ICategoryManager, CategoryManager>();
+            services.AddTransient<IConverter<CategoryDTO, CategoryModel>, CategoryConverter>();            
```

Comment:
Group by managers, converters for better support

Reply:
done